### ISSUE TITLE
[Bug] Cheats not working

### ISSUE THREAD

**Issue by dfp2020** [CorsixTH/CorsixTH#2102]
#### Describe the issue
the cheats not working after version 0.65, it worked in the past

#### Steps to Reproduce
1. *Load save game attached to this ticket*

2.

3.

#### System Information
**CorsixTH Version:** *e.g. 0.50 or 7d886f35ca*

Operating System: Windows 11, build 21H2, 22000,526

Theme Hospital Version: Origin, 

#### Gamelog.txt

gamelog.txt and saves 
[bugcorsixth.zip](https://github.com/CorsixTH/CorsixTH/files/8072137/bugcorsixth.zip)

```
Gamelog.txt
Created game with savegame version 127.
Warning: Trying to remove nonexistant room build callback (function: 00000222B4332760) from humanoid (table: 00000222B1739C60).
Warning: Trying to remove nonexistant room remove callback (function: 00000222AC24C5D0) from humanoid (table: 00000222B1739C60).
Warning: Trying to remove nonexistant room build callback (function: 00000222AC9601A0) from humanoid (table: 00000222AA8E7370).
Warning: Trying to remove nonexistant room remove callback (function: 00000222B0842620) from humanoid (table: 00000222AA8E7370).
Warning: Trying to remove nonexistant room build callback (function: 00000222A39E1CB0) from humanoid (table: 00000222B2BEDEA0).
Warning: Trying to remove nonexistant room remove callback (function: 00000222A3EDA0A0) from humanoid (table: 00000222B2BEDEA0).
Warning: Trying to remove nonexistant room build callback (function: 00000222B16E75D0) from humanoid (table: 00000222ABBD2710).
Warning: Trying to remove nonexistant room remove callback (function: 00000222AB0CB240) from humanoid (table: 00000222ABBD2710).
Savegame version is 127 (v0.62), originally it was 127 (v0.62)
Warning: Trying to remove nonexistant room build callback (function: 000001EA5898C7B0) from humanoid (table: 000001EA624B7020).
Warning: Trying to remove nonexistant room remove callback (function: 000001EA584F6B80) from humanoid (table: 000001EA624B7020).
Savegame version is 127 (v0.62), originally it was 127 (v0.62)
Savegame version is 127 (v0.62), originally it was 127 (v0.62)
Savegame version changed from 127 (v0.62) to 134 (v0.63). The save was created using 127 (v0.62)
Error in timer handler: 
Lua\humanoid_actions\walk.lua:335: Humanoid (Stripped Female Patient 2) without door animations trying to walk through door (from cardiogram to corridor)
stack traceback:
	[C]: in function 'error'
	Lua\humanoid_actions\walk.lua:335: in function <Lua\humanoid_actions\walk.lua:255>
	(...tail calls...)
	Lua\humanoid_actions\walk.lua:233: in local 'timer_function'
	Lua\entity.lua:216: in method 'tick'
	Lua\world.lua:1042: in method 'onTick'
	Lua\app.lua:1111: in function <Lua\app.lua:1108>
	(...tail calls...)
	Lua\app.lua:1012: in function <Lua\app.lua:1007>
```

### Additional Info
*Paste any screen shots or other additional information that might help
illustrate the problem.*

**Comment by dfp2020** [comment]
the cheats not working after version 0.66, it worked in the past

can't find game.txt or my saved games

Operating System: e.g. Windows 11, build 21H2, 22000,526

Theme Hospital Version: Origin,

**Comment by dfp2020** [comment]
Gamelog.txt
Created game with savegame version 127.
Warning: Trying to remove nonexistant room build callback (function: 00000222B4332760) from humanoid (table: 00000222B1739C60).
Warning: Trying to remove nonexistant room remove callback (function: 00000222AC24C5D0) from humanoid (table: 00000222B1739C60).
Warning: Trying to remove nonexistant room build callback (function: 00000222AC9601A0) from humanoid (table: 00000222AA8E7370).
Warning: Trying to remove nonexistant room remove callback (function: 00000222B0842620) from humanoid (table: 00000222AA8E7370).
Warning: Trying to remove nonexistant room build callback (function: 00000222A39E1CB0) from humanoid (table: 00000222B2BEDEA0).
Warning: Trying to remove nonexistant room remove callback (function: 00000222A3EDA0A0) from humanoid (table: 00000222B2BEDEA0).
Warning: Trying to remove nonexistant room build callback (function: 00000222B16E75D0) from humanoid (table: 00000222ABBD2710).
Warning: Trying to remove nonexistant room remove callback (function: 00000222AB0CB240) from humanoid (table: 00000222ABBD2710).
Savegame version is 127 (v0.62), originally it was 127 (v0.62)
Warning: Trying to remove nonexistant room build callback (function: 000001EA5898C7B0) from humanoid (table: 000001EA624B7020).
Warning: Trying to remove nonexistant room remove callback (function: 000001EA584F6B80) from humanoid (table: 000001EA624B7020).
Savegame version is 127 (v0.62), originally it was 127 (v0.62)
Savegame version is 127 (v0.62), originally it was 127 (v0.62)
Savegame version changed from 127 (v0.62) to 134 (v0.63). The save was created using 127 (v0.62)
Error in timer handler: 
Lua\humanoid_actions\walk.lua:335: Humanoid (Stripped Female Patient 2) without door animations trying to walk through door (from cardiogram to corridor)
stack traceback:
	[C]: in function 'error'
	Lua\humanoid_actions\walk.lua:335: in function <Lua\humanoid_actions\walk.lua:255>
	(...tail calls...)
	Lua\humanoid_actions\walk.lua:233: in local 'timer_function'
	Lua\entity.lua:216: in method 'tick'
	Lua\world.lua:1042: in method 'onTick'
	Lua\app.lua:1111: in function <Lua\app.lua:1108>
	(...tail calls...)
	Lua\app.lua:1012: in function <Lua\app.lua:1007>

**Comment by lewri** [comment]
According to the gamelog you're using `0.63`?
`Savegame version changed from 127 (v0.62) to 134 (v0.63). The save was created using 127 (v0.62)`

Though the gamelog error doesn't add up to your experienced problem.

**Comment by dfp2020** [comment]
no i am using the latest version

**Comment by dfp2020** [comment]
![corsixth_intro](https://user-images.githubusercontent.com/70529432/154110876-286d49c8-cd60-42e1-aa54-7e7853d958d9.png)

**Comment by lewri** [comment]
I think I know why. If you close the fax can you use the cheats?

**Comment by dfp2020** [comment]
oh i have not think about, i give it a try

**Comment by dfp2020** [comment]
It works, thank you Lewri

**Comment by lewri** [comment]
Thanks. I'll need to reopen to fix the actual issue for 0.66
